### DISCUSSION TITLE
Tune Menu Question

### DISCUSSION THREAD

**Discussion by bmizes** [mriscoc/Ender3V2S1#50] [Q&A] [answered]
I have a question on the Speed parameter of the Tune menu in terms of exactly what this does.  If I want to slow down the print speed at some point in the print to improve the print quality, is the speed parameter the appropriate one to tune or is this adjusting something else?  If this is not the correct parameter, is there anything in the tune menu I can adjust to reduce the print speed?  Thanks.

**Comment by duncio** [comment, marked as answer]
Yes, that is the correct thing to use if you want to reduce the printing speed.
